articles/openstack/ostack-how-use-octavia.md

```diff
@@ -0,0 +1,240 @@
+author: bbramble
+reviewer: 
+lastreviewed: 12/08/2021
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use Octavia Load Balancing as a Service
+toc_fullpath: How To/ostack-how-use-octavia.md
+toc_mdlink: ostack-how-use-octavia.md
+---
+
+# How to use Octavia Load Balancing as a Service on UKCloud for OpenStack
+
+> [!NOTE]
+> This article does not apply to OpenStack regions running OpenStack Platform 10 (OSP10) and lower (currently COR00005).
+
+## Overview
+
+Octavia is the OpenStack Load Balancing as a Service (LBaaS) solution and forms part of the solution for availability and scaling that modern systems require.
+
+Octavia currently supports a number of configurations:
+
+- TCP, HTTP, HTTPS, terminated HTTP and UDP support
+
+- Source IP, round robin and least connections balancing
+
+- Health monitoring using ping, HTTP, TLS Hello and UDP
+
+## Prerequisites
+
+If you intend to manage with the OpenStackCLI, you should consider the following:
+
+- Octavia uses port 13876 for communication. This port will need to be open to manage Octavia.
+
+- You must be using a recent version of the OpenStackCLI.
+
+## Use cases
+
+Use cases for Octavia include:
+
+- Scaling out website traffic by balancing it over multiple servers
+
+- Increasing the availability of a service with active-standby or active-active pools
+
+## Octavia components
+
+Octavia consists of a number of service components that need to be configured as follows:
+
+- **Load balancer.** The load balancer sits at the top of the Octavia component stack. The Octavia load balancer connects to an OpenStack network and provides the entry point for traffic. You may assign a load balancer a floating IP to connect it to internet traffic.
+
+- **Listener.** An Octavia listener defines an IP protocol and service that the load balancer will accept. This can be TCP, UDP, HTTP or HTTPS (direct or terminated). Each load balancer may have multiple listeners to support more than one service (for example, HTTP and HTTPS).
+
+- **Pool.** An Octavia pool defines a list of servers that will accept the traffic from the listener. As with the listener, you select the protocol for the pool to use.
+
+- **Member.** Each Octavia pool needs to contain a number of members. These specify the IP address and port of the servers (or services) that will be used to service the requests. You can also define a separate monitoring IP and port distinct from the request ones if needed.
+
+- **Health monitor.** The health monitor defines how Octavia can check that the member resources are online and available, automatically removing or re-adding from the list of members in a pool should it be required. The type of check performed is configurable, as is the frequency of checks, timeout and number of failed (or successful) checks required for the status of a member to be changed.
+
+## Examples
+
+### Assumptions
+
+The requirement is to create an HTTP load balancer across existing servers.
```

Comment:
Feels like there is something missing here. Possibly add something like:

The following must exist in your environment: